pkg/stanza/docs/operators/journald_input.md

```diff
@@ -33,7 +34,23 @@ The `journald_input` operator will use the `__REALTIME_TIMESTAMP` field of the j
 - type: journald_input
   priority: emerg..err
 ```
-#### Simple journald input
+
+#### Matches
+
+The following configuration:
+
+```yaml
+- type: journald_input
+  matches:
+    - _SYSTEMD_UNIT: ssh
+    - _SYSTEMD_UNIT: kubelet
+      _UID: "1000"
```

Comment:
Are these AND'd or OR'd together? Seems worth noting in the readme.

Reply:
These are `AND`ed. I will update doc